sycl/doc/extensions/supported/sycl_ext_oneapi_queue_priority.asciidoc

```diff
@@ -0,0 +1,103 @@
+= sycl_ext_oneapi_queue_priority
+
+:source-highlighter: coderay
+:coderay-linenums-mode: table
+
+// This section needs to be after the document title.
+:doctype: book
+:toc2:
+:toc: left
+:encoding: utf-8
+:lang: en
+:dpcpp: pass:[DPC++]
+
+// Set the default source code type in this document to C++,
+// for syntax highlighting purposes.  This is needed because
+// docbook uses c++ and html5 uses cpp.
+:language: {basebackend@docbook:c++:cpp}
+
+
+== Notice
+
+[%hardbreaks]
+Copyright (C) 2022-2022 Intel Corporation.  All rights reserved.
+
+Khronos(R) is a registered trademark and SYCL(TM) and SPIR(TM) are trademarks
+of The Khronos Group Inc.  OpenCL(TM) is a trademark of Apple Inc. used by
+permission by Khronos.
+
+== Contact
+
+To report problems with this extension, please open a new issue at:
+
+https://github.com/intel/llvm/issues
+
+
+== Dependencies
+
+This extension is written against the SYCL 2020 revision 4 specification.  All
+references below to the "core SYCL specification" or to section numbers in the
+SYCL specification refer to that revision.
+
+== Status
+
+This extension is implemented and fully supported by {dpcpp}.
+Although {dpcpp} supports this extension on all backends, it is currently used
+only on Level Zero. Other backends ignore this new property.
```

Comment:
```suggestion
[NOTE]
====
Although {dpcpp} supports this extension on all backends, it is currently used
only on Level Zero. Other backends ignore the properties defined in this specification.
====
```

Our convention is to highlight implementation limitations as "NOTE" callouts.  I think the suggestion above will do this, but check it in the rendered HTML.

Reply:
OK